apisix/stream/xrpc/runner.lua

```diff
@@ -91,7 +96,7 @@ end
 
 local function finialize_req(protocol, session, ctx)
     ctx._rpc_end_time = ngx_now()
-
+    session._curr_ctx = ctx
```

Comment:
We don't need it now?

Reply:
My mistake, I forgot to delete it. Deleted.